packages/visx-responsive/src/index.ts

```diff
@@ -1,4 +1,6 @@
-export { default as ScaleSVG } from './components/ScaleSVG';
 export { default as ParentSize } from './components/ParentSize';
-export { default as withParentSize } from './enhancers/withParentSize';
-export { default as withScreenSize } from './enhancers/withScreenSize';
+export { default as ScaleSVG } from './components/ScaleSVG';
+export { default as withParentSize, WithParentSizeProvidedProps } from './enhancers/withParentSize';
+export { default as withScreenSize, WithScreenSizeProvidedProps } from './enhancers/withScreenSize';
+export { default as useParentSize, UseParentSizeConfig } from './hooks/useParentSize';
+export { default as useScreenSize, UseScreenSizeConfig } from './hooks/useScreenSize';
```

Comment:
These lines seem to be causing an error in TypeScript (if you have it scan node_modules) because those new types are not exported from the transpiled versions of these 4 files.

Reply:
Ah damn. Sorry about that. Created a PR that should fix this issue.

Reply:
oof, thank you. also found a blind spot in our CI, thanks for the quick report and fix 🙏